### PR TITLE
refactor(frontend): New descriptive hyperlink ESP #523

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- resolves #523

### Steps to test
1. Go to VSCode and search "enlace]", "información]", aquí]", guía]"

#### CheckList
- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Review by warnerHurtado — COMMENTED** [review]
LGTM

File: docs/aprender-eosio/funciones-importantes.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Por lo general, esta autoridad se invoca mediante un contrato de firma múltiple
...
+Los contratos inteligentes de EOSIO son simples archivos de WebAssembly. El formato WebAssembly permite definir importaciones, funciones que no se declaran en WASM en sí pero en el entorno en el que se ejecuta el código de WebAssembly. EOSIO utiliza estas funciones integradas (intrínsecas) para intercambiar datos entre el código de contrato de WebAssembly y el nodo EOSIO `nodeos`. Cada vez que la máquina virtual EOS (EOSVM), que ejecuta el código de WebAssembly, llega a un nivel intrínseco, el control del programa se transfiere al código del controlador de nodo para resolverlo. Una lista completa de funciones intrínsecas para EOSIO 2.0.X se puede encontrar [Funciones Intrínsecas EOSIO](https://github.com/EOSIO/eosio.cdt/blob/a6b8d3fc289d46f4612588cdd7223a3d549238f6/libraries/native/native/eosio/intrinsics_def.hpp#L42-L160).
```
Comment (xavier506) [inline docs/aprender-eosio/funciones-importantes.md:17]: encontrar  en el siguiente enlace: [Funciones Intrínsecas EOSIO]

Comment (JAMares) [reply docs/aprender-eosio/funciones-importantes.md]: Fixed up

**Review by xavier506 — CHANGES_REQUESTED** [review]
view comments

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**